app/wp-content/themes/kstg/index.php

```diff
@@ -1,76 +1,49 @@
-                    							); ?>
-									</p>
-
-								</header>
-
-								<section class="entry-content cf">
-									<?php the_content(); ?>
-								</section>
-
-								<footer class="article-footer cf">
-									<p class="footer-comment-count">
-										<?php comments_number( __( '<span>No</span> Comments', 'bonestheme' ), __( '<span>One</span> Comment', 'bonestheme' ), __( '<span>%</span> Comments', 'bonestheme' ) );?>
-									</p>
-
-
-                 	<?php printf( '<p class="footer-category">' . __('filed under', 'bonestheme' ) . ': %1$s</p>' , get_the_category_list(', ') ); ?>
-
-                  <?php the_tags( '<p class="footer-tags tags"><span class="tags-title">' . __( 'Tags:', 'bonestheme' ) . '</span> ', ', ', '</p>' ); ?>
-
-
-								</footer>
-
-							</article>
-
-							<?php endwhile; ?>
-
-									<?php bones_page_navi(); ?>
-
-							<?php else : ?>
-
-									<article id="post-not-found" class="hentry cf">
-											<header class="article-header">
-												<h1><?php _e( 'Oops, Post Not Found!', 'bonestheme' ); ?></h1>
-										</header>
-											<section class="entry-content">
-												<p><?php _e( 'Uh Oh. Something is missing. Try double checking things.', 'bonestheme' ); ?></p>
-										</section>
-										<footer class="article-footer">
-												<p><?php _e( 'This is the error message in the index.php template.', 'bonestheme' ); ?></p>
-										</footer>
-									</article>
-
-							<?php endif; ?>
-
-
-						</main>
-
-					<?php get_sidebar(); ?>
-
-				</div>
-
+<div class="homepage-wrapper">
+	<section class="main">
+		<div class="gallery-container row">
+			<div class="gallery-corporate col-1-4">
+				<img class="fluid-images" src="<?php bloginfo('template_url'); ?>/library/images/corporate.png" alt="corporate services" width="280" height="350" />
+				<a href=""><p>CORPORATE &#8594;</p></a>
 			</div>
-
+			<div class="gallery-events col-1-4">
+				<img class="fluid-images" src="<?php bloginfo('template_url'); ?>/library/images/personal.png" alt="personal services and events" width="280" height="350" />
+				<a href=""><p class="events">PERSONAL & EVENTS &#8594;</p></a>
+			</div>
+			<div class="gallery-travel col-1-4">
+				<img class="fluid-images" src="<?php bloginfo('template_url'); ?>/library/images/travel.png" alt="travel services" width="280" height="350" />
+				<a href=""><p>TRAVEL &#8594;</p></a>
+			</div>
+			<div class="gallery-wedding col-1-4">
+				<img class="fluid-images" src="<?php bloginfo('template_url'); ?>/library/images/wedding.png" alt="personal services and events" alt="wedding services" width="280" height="350" />
+				<a href=""><p>WEDDING &#8594;</p></a>
+			</div>
+		</div>
+		<div class="about">
+			<h1>The Philadelphia Region's Newest Luxury Transportation Company</h1>
```

Comment:
Maybe once you get everything built, it'd be nice to go back and make this dynamic. We aren't doing any sort of support after it's live, and I'd imagine he might want to change this at some point in the future without getting a developer. I can help you with that (hopefully we can make use of the custom post plugin we learned about recently)